### DISCUSSION TITLE
differentiate between  rows

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#9433] [Help] [answered]
Hello everoyne.
Does anyone have an idea to differentiate between the rows of a table (UI) according to the value of a column
for example if (stock==0) do something on the row and if (stock !=0 ) do something else.

@melohagan @andz-bb @shogunpurple 

Thank you.

**Comment by melohagan** [comment, marked as answer]
What is the something you are trying to do? Screenshots would be helpful

**Reply by rafikApp** [reply]
Hi @melohagan 
at first I wanted to change the color according to the value of the stock, but since that is not possible is there another idea to differentiate between the rows (add an icon or something like that)

**Reply by melohagan** [reply]
One thing I have done in the past was to add a [Tag](https://docs.budibase.com/docs/tag) to a table component. Then I used [Conditional UI](https://docs.budibase.com/docs/conditions) to set the colour depending on some row column state.

![Screenshot 2023-01-26 at 09 27 10](https://user-images.githubusercontent.com/101575380/214801349-ca9272a2-6a49-4f5c-bd56-077357bad42f.png)

![Screenshot 2023-01-26 at 09 27 41](https://user-images.githubusercontent.com/101575380/214801461-40e8a9b2-3713-44d4-a3a0-2b23e492f786.png)

**Reply by rafikApp** [reply]
Hi @melohagan 
I tried this but it doesn't work 
{{ CONSIGNATIONS.Selected rows.statut }}


![Capture](https://user-images.githubusercontent.com/106697208/214817950-b6dd9454-5445-44cd-b107-d47e95aaabcd.PNG)

**Reply by rafikApp** [reply]
It's ok @melohagan Thank you.

**Reply by melohagan** [reply]
It wouldn't be selected rows. It would just be `<table-component-name>.<data-table-name>.statut`.

You can use the bindings helper to make sure the binding is right:

![Screenshot 2023-01-26 at 11 10 53](https://user-images.githubusercontent.com/101575380/214821755-d5d241fd-4077-4553-a8fd-978cf247fc68.png)

![Screenshot 2023-01-26 at 11 11 27](https://user-images.githubusercontent.com/101575380/214821860-d0fd9b05-b4c5-4bda-b3e0-990f6e5b3fbe.png)

But in your case it would be 'statut' instead of 'City'.

**Reply by rafikApp** [reply]
@melohagan 
how can i eliminate the spaces?, so that it is visible.
I want it to cover the whole cell if possible
![tag](https://user-images.githubusercontent.com/106697208/214825633-f71980c2-f478-48ea-baea-a11b5b04c8c7.PNG)

**Reply by melohagan** [reply]
You'll most likely need to use some global styling: https://docs.budibase.com/docs/custom-css#global-styling

Right click > Inspect to get the names of the div classes you'll need to work with and go from there.
Sorry there isn't a more convenient way to go about it for now.
